### PR TITLE
Update tic-tac-toe-new-ui.py

### PR DESCRIPTION
**Pull request by harshitashyale**
Made some minor fixes

### CODE REVIEW

**Comment by githubparticipant** [comment]
Seems like there's no score updating logic?

**Comment by harshitashyale** [comment]
That's for next release

File: tic-tac-toe-new-ui.py [hunk 1]
```diff
@@ -19,14 +19,14 @@ def printBoard(playerY, playerX):
     print(f"| {six} | {seven} | {eight} |") 
 
 
-def checkwinner(playerX, playerY):
+def checkwinner(playerY, playerX):
```
Comment (githubparticipant) [inline tic-tac-toe-new-ui.py:22]: switch variables around

Comment (harshitashyale) [reply tic-tac-toe-new-ui.py:22]: do you mean (playerX, playerY)?

**Comment by githubparticipant** [comment]
make it simpler

File: tic-tac-toe-new-ui.py (hunk 1 above)
Comment (githubparticipant) [reply tic-tac-toe-new-ui.py:22]: yes

File: tic-tac-toe-new-ui.py [hunk 2]
```diff
@@ -19,14 +19,14 @@ def printBoard(playerY, playerX):
...
     wins = [[0, 1, 2], [3, 4, 5], [6, 7, 8],[0, 3, 6], [1, 4, 7], [2, 5, 8], [0, 4, 8], [2, 4, 6]]
     for win in wins:
         if(sum(playerX[win[0]], playerX[win[1]], playerX[win[2]]) == 3):
-            print("Player Y has won the game")
+            print("Player X has won the game")
             return 1
         if(sum(playerY[win[0]], playerY[win[1]], playerY[win[2]]) == 3):
```
Comment (githubparticipant) [inline tic-tac-toe-new-ui.py:28]: recheck logic

File: tic-tac-toe-new-ui.py [hunk 3]
```diff
@@ -43,7 +43,7 @@ def checkwinner(playerX, playerY):
             value = int(input("Enter your Value: "))
             playerX[value] = 1
         else:
```
Comment (githubparticipant) [inline tic-tac-toe-new-ui.py:45]: recheck

File: tic-tac-toe-new-ui.py [hunk 4]
```diff
@@ -43,7 +43,7 @@ def checkwinner(playerX, playerY):
...
-            print("O's Chance to Play")
+            print("Y's Chance to Play")
             value = int(input("Enter your Value: "))
             playerY[value] = 1
```
Comment (githubparticipant) [inline tic-tac-toe-new-ui.py:48]:
```suggestion
            playerY[value] = 0
```

**Comment by githubparticipant** [comment]
Make this simpler

File: tic-tac-toe-new-ui.py (hunk 1 above)
Comment (githubparticipant) [reply tic-tac-toe-new-ui.py:22]: Yes

File: tic-tac-toe-new-ui.py (hunk 4 above)
Comment (githubparticipant) [inline tic-tac-toe-new-ui.py:48]: Check this as well

Comment (githubparticipant) [inline tic-tac-toe-new-ui.py:48]:
```suggestion
            playerY[value] = 0
```

**Comment by githubparticipant** [comment]
Please make the code simpler.

File: tic-tac-toe-new-ui.py (hunk 1 above)
Comment (githubparticipant) [reply tic-tac-toe-new-ui.py:22]: yes

**Comment by wuguishifu** [comment]
hi jenn

**Pull request closed without merging**